.github/workflows/docker.yml

```diff
@@ -0,0 +1,204 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - main
+      - develop
+      - 'release/*'
+
+    tags:
+      - '*'
+
+  pull_request:
+
+env:
+  MAIN_REPO: ncsa/standalone-smm-analytics
+
+jobs:
+
+  # ----------------------------------------------------------------------
+  # DOCKER BUILD
+  # ----------------------------------------------------------------------
+  docker:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+      matrix:
+        name:
+          - autophrase
+          - histogram
+          - check_screen_name
+          - classification_predict
+          - classification_train
+          - classification_split
+          - collect_reddit_comment
+          - crimson_hexagon_monitors
+          - image_crawler
+          - name_entity_recognition
+          - network_analysis
+          - preprocessing
+          - screen_name_prompt
+          - sentiment_analysis
+          - topic_modeling
```

Comment:
Could we add all the clowder ones as well: https://github.com/ncsa/standalone-smm-analytics/tree/develop/containerized_analytics/clowder